tests/meta/compress/test_clean_fastq.py

```diff
@@ -95,6 +93,9 @@ def test_update_hk_fastq(
     )
     for spring_file in [hk_spring_files, hk_spring_metadata_files]:
         assert spring_file
+        for tag_name in [tag.name for tag in fastq[run]["hk_first"].tags]:
```

Comment:
I don't understand this line. Shouldn't it be `spring_file.tags`?

Reply:
@diitaz93 I wanted to make sure that all tags from the fastq file was passed on, so I loop through the fastq tags and ensure that the spring (metadata) file is tagged with them.

Reply:
Is clear now 👍